src/witan/models/run_models.clj

```diff
@@ -102,10 +103,28 @@
       (data-csv/write-csv out-file
                           (convert-row-maps-to-vector-of-vectors ordered-data)))))
 
+(def local-authorities (read-string (slurp "resources/default_datasets/local_authorities.edn")))
+
+(defn get-district [gss-code]
+  (get-in local-authorities [:gss-code-to-district (keyword gss-code)]))
+
+(defn add-district-to-dataset [data] (wds/add-derived-column data :district [:gss-code] (fn [gss-code] (get-district gss-code))))
+
+(defn add-district-to-dataset2 [data gss-code] (ds/add-column data :district (repeat (get-district gss-code))))
+
 (defn run-ccm
   "This function will evolve as we build the jar.
    Takes in a map of datasets, a gss code and a map of parameters.
    Returns a dataset with historical data and projections."
   [inputs gss-code params]
   (core/looping-test (get-datasets inputs gss-code)
                      params))
+
+(defn output-ccm
+  "runs the ccm, adds district information for easier human
+   reading and outputs to a csv"
+  [inputs gss-code params filepath]
+  (prn (str "Preparing projection for " (get-district gss-code) "..."))
```

Comment:
It's just bikeshedding, but I remember Bruce often recommend the use of `format` over `str` (https://clojuredocs.org/clojure.core/format).